src/agent/onefuzz-agent/src/local/tui.rs

```diff
@@ -221,6 +306,9 @@ impl TerminalUi {
         let file_count = ui_state.file_count;
         let mut log_event_receiver = ui_state.log_event_receiver;
         let mut terminal = ui_state.terminal;
+        let rate = ui_state.coverage.rate.unwrap_or(0.0);
+        let features = ui_state.coverage.features.unwrap_or(0);
+        let covered = ui_state.coverage.covered.unwrap_or(0);
```

Comment:
It appears the coverage sub-window always shows up, even if I don't have coverage enabled.

As an example, I would expect this command to not show the coverage window:

`onefuzz-agent local libfuzzer-fuzz --crashes_dir /tmp/testx/crashes/ --inputs_dir /tmp/testx/inputs/ --target_exe ~/projects/onefuzz/test-artifacts/linux-libfuzzer/fuzz.exe`